docs/content/drafts/ActionMenu2.mdx

```diff
@@ -0,0 +1,251 @@
+---
+title: ActionMenu
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/ActionMenu
+storybook: '/react/storybook?path=/story/composite-components-actionmenu2'
+description: An ActionMenu is an ActionList-based component for creating a menu of actions that expands through a trigger button.
+---
+
+import {BorderBox, Avatar} from '@primer/components'
+import {ActionMenu, ActionList} from '@primer/components/drafts'
+import {Props} from '../../src/props'
+
+<br />
+
+<BorderBox sx={{padding: 6}}>
```

Comment:
FYI, BorderBox is deprecated. Can we use Box here instead?

Reply:
Good point! Fixed